tests/pfcwd/test_pfcwd_cli.py

```diff
@@ -487,8 +499,6 @@ def test_pfcwd_show_stat(self, request, setup_pfc_test, setup_dut_test_params, e
                 logger.info("Pfcwd action {} on port {}: Tx traffic action {}, Rx traffic action {} ".
                             format(action, port, self.tx_action, self.rx_action))
                 self.run_test(self.dut, port, action)
-            except Exception as e:
```

Comment:
why remove the exception?

Reply:
There's no reason to catch the exception just to rethrow it via "pytest.fail(str(e))".
The "try-finally" pattern can be used without the "except" clause, this makes it so the "finally" code will always execute even if an exception occurs.
Rethrowing obfuscates the real location of the exception, making quick log-parsing and triage much more difficult when browsing a short summary. 
There's a number of these that need to be removed, I recently removed some of them over in a different PFCWD file.